fastcov.py

```diff
@@ -30,6 +30,7 @@
 import threading
 import subprocess
 import multiprocessing
+from pathlib import Path
```

Comment:
Any compelling reason not to just use `os.path.split` to avoid another dependency?

Reply:
os.path.split only does a head and tail split. The intent is to have all directory levels into an array.
Meaning:
/home/lapointejp/git/project would turn into the array ['/', 'home', 'lapointejp', 'git', 'project'].

I don't think os.path provides this functionality ?

Reply:
I see what you are saying. You could do just a normal split with the os path separator:

```python
parts = path.split(os.path.sep)
```

which would correctly split `"/a/b/c"` into 4 parts, and `a/b/c` into 3. But, I suppose if you've already tested using `Path`, then it doesn't hurt to use `pathlib`.